runner/views/run_api_view.py

```diff
@@ -652,6 +653,82 @@ def post(self, request):
         return Response(body, status=status.HTTP_202_ACCEPTED)
 
 
+class ArgosMappingViewSet(GenericAPIView):
+    logger = logging.getLogger(__name__)
+
+    serializer_class = ArgosPairingSerializer
+
+    def post(self, request):
+        igo_request_id = request.data.get("igo_request_id", None)
+        argos_slug = request.data.get("argos_slug", None)
+        if igo_request_id and argos_slug:
+            operator_model = Operator.objects.get(slug=argos_slug)
+            operator = OperatorFactory.get_by_model(operator_model, request_id=igo_request_id)
+            # construct_argos_jobs() is sloppily separate from the Operator module
+            from runner.operator.argos_operator.v2_0_0.construct_argos_pair import construct_argos_jobs
```

Comment:
Is this import here on purpose? Should we move it on top of the file with other imports?

Reply:
The problem with these is they're tied to the argos operator and that specific version

It's really messy, I'd prefer to not have it up top for now